src/test/java/nextstep/subway/unit/LineTest.java

```diff
@@ -137,4 +125,52 @@ void addSectionWithNotExistingStations() {
       assertThat(thrown).isInstanceOf(IllegalArgumentException.class);
     }
   }
+  @Nested
+  @DisplayName("도메인 (Line): 구간 삭제할 때")
+  class SectionRemovingTest {
+    @DisplayName("도메인(Line): 구간의 마지막 역을 삭제")
+    @Test
+    void removeSection() {
+      //Given
+      첫번째노선.addSection(Section.of(첫번째노선, 10L, 두번째역, 세번째역));
+      //When
+      첫번째노선.deleteSection(세번째역);
+      //Then
+      assertThat(첫번째노선.getStations().stream().map(station -> station.getName())).containsExactly(
+          첫번째역.getName(), 두번째역.getName());
+    }
+
+    @DisplayName("도메인(Line): 구간의 중간 역을 삭제")
+    @Test
+    void removeMiddleSection() {
+      //Given
+      첫번째노선.addSection(Section.of(첫번째노선, 10L, 두번째역, 세번째역));
+      //When
+      첫번째노선.deleteSection(두번째역);
+      //Then
+      assertThat(첫번째노선.getStations().stream().map(station -> station.getName())).containsExactly(
```

Comment:
```suggestion
      assertThat(첫번째노선.getStations().stream().map(station -> station.getName()))
          .containsExactly(첫번째역.getName(), 세번째역.getName());
```

[Google Style Guide](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break) 가 일반적으로 많이 따르는 스타일인 것 같아요
[객체지향생활체조](https://edu.nextstep.camp/s/c30VOHNP/ls/8tTBk5Yp) 에서도 `규칙 4: 한 줄에 점을 하나만 찍는다.` 와 같이 줄바꿈의 기준을 (.) 으로 많이 합니다 😅

Reply:
그렇군요~~ 사실 Google Style Guide를 적용한 채로 포맷팅을 다잡고 돌려서 해당 결과가 나왔는데
operator 기준으로 의식적으로 잘라 보겠습니다!~